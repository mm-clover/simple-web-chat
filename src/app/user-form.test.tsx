import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom'
import UserFormDialog from './user-form';


describe('User Form', () => {
  it('should render user name input', () => {
    render(<UserFormDialog />);
    const inputEl = screen.getByTestId("username-input");
    expect(inputEl).toBeInTheDocument();
  });
});