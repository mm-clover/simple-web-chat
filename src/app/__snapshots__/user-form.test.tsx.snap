// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Form should render and match the snapshot 1`] = `
<React.Fragment>
  <ForwardRef(Dialog)
    fullWidth={true}
    maxWidth="sm"
    onClose={[Function]}
    open={true}
  >
    <ForwardRef(DialogTitle)>
      Please enter your name!
    </ForwardRef(DialogTitle)>
    <ForwardRef(DialogContent)>
      <ForwardRef(TextField)
        autoFocus={true}
        fullWidth={true}
        id="name"
        label="Your Name"
        margin="dense"
        onChange={[Function]}
        onKeyDown={[Function]}
        type="name"
        variant="standard"
      />
    </ForwardRef(DialogContent)>
    <ForwardRef(DialogActions)>
      <ForwardRef(Button)
        onClick={[Function]}
      >
        Cancel
      </ForwardRef(Button)>
      <ForwardRef(Button)
        onClick={[Function]}
      >
        Enter to the chart
      </ForwardRef(Button)>
    </ForwardRef(DialogActions)>
  </ForwardRef(Dialog)>
  
</React.Fragment>
`;
